app/controllers/contact_messages_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ContactMessagesController < ApplicationController
+  def new
+    @contact_message = ContactMessage.new
+  end
+
+  def create
+    @contact_message = ContactMessage.new(contact_message_params)
+    if @contact_message.save
+      NewContactMessageMailer.contact_message_notification(
+        @contact_message).deliver_now
+      flash[:success] =
+        "It's good to hear from you. We will respond to your message shortly."
+      redirect_to root_path
+    else
+      flash.now[:error] =
+        "Please review your message, some information is missing."
+      render :new
+    end
+  end
+
+  private
+
+  def contact_message_params
+    params.require(:contact_message).permit(
+      :name,
+      :email,
+      :cell,
+      :message
```

Comment:
I try and do it from the beginning, but I do agree that it creates unnecessary and possibly confusing noise in a PR where you decide to alphabetize everything. The organization benefit for me outweighs the other stuff. It's more of a "cooking in a clean kitchen" thing to me I think.